README.md

```diff
@@ -1,2 +1,33 @@
-# COMP4350-Project
+# ![logo](assets/Logo2.png)BisonCoin
+
 Repo containing documents and code for our COMP 4350 Group Project.
+
+## Members
+
+- Abhi Sachdev ([GitHub](https://github.com/abhisachdev17))
+- Madison Fines ([GitHub](https://github.com/madisonfines))
+- Akshay Sharma ([GitHub](https://github.com/akshaysharma21))
+- Zack Holmberg ([GitHub](https://github.com/ZackHolmberg))
+
+## Project Description
+
+- BisonCoin is a new cryptocurrency intended for use by University of Manitoba students to exchange on campus between each other or a variety of products at all campus businesses, including all food outlets and the book store!
+
+- The system will be composed of:
+  - A front-end web application
+  - A collection of microservices
+  - A Blockchain, cryptocurrency implementation
+  - Data storage
+
+## Core Features
+
+- Students will be able to send and receive BisonCoin to their peers, as well as businesses in order to purchase products.
+- Students will be able to use the app to view how much BisonCoin they have in their digital wallet, as well as view their transaction history.
+- Students will be able to act as miners on the blockchain network, earning currency as a reward.
+
+## Tech Stack
+
+- **Blockchain:** `Python`
+- **Microservices:** `Python`
```

Comment:
Flask in here

Reply:
Flask is just a Python library so I don't know if we need to specify it, that's my thoughts at least. Lmk what you think

Reply:
Wouldn't hurt to specify, example repo specified Node and Express as well.